Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -105,8 +105,8 @@
 		16F52ECC6FA8A0587CD779EB /* user_test.cc in Sources */ = {isa = PBXBuildFile; fileRef = AB38D93220239654000A432D /* user_test.cc */; };
 		16FE432587C1B40AF08613D2 /* objc_type_traits_apple_test.mm in Sources */ = {isa = PBXBuildFile; fileRef = 2A0CF41BA5AED6049B0BEB2C /* objc_type_traits_apple_test.mm */; };
 		1733601ECCEA33E730DEAF45 /* autoid_test.cc in Sources */ = {isa = PBXBuildFile; fileRef = 54740A521FC913E500713A1A /* autoid_test.cc */; };
+		175718BEAB9AC0951FF0BFC1 /* watch_change_test.cc in Sources */ = {isa = PBXBuildFile; fileRef = 622E45CDB5D5FE6307155010 /* watch_change_test.cc */; };
```

Comment:
You'll need to update the cmake files too.

Oh, and as a side note, on my compiler, int64_t == long. So:
```
testutil.h:94:26: error: redefinition of ‘firebase::firestore::model::FieldValue firebase::firestore::testutil::Value(int64_t)’
 inline model::FieldValue Value(int64_t value) {
                          ^~~~~
testutil.h:90:26: note: ‘firebase::firestore::model::FieldValue firebase::firestore::testutil::Value(long int)’ previously defined here
 inline model::FieldValue Value(long value) {  // NOLINT(runtime/int)
```

(That's not currently in master, but somewhere in this chain of commits.)

Reply:
Done. The Value fix is more involved and isn't really related to this at all so I've created #3600 to handle that.